python/tvm/contrib/rocm.py

```diff
@@ -0,0 +1,51 @@
+"""Utility for ROCm backend"""
+import sys
+import subprocess
+from . import util
+from ..api import register_func
+
+def rocm_link(in_file, out_file):
+    """Link relocatable ELF object to shared ELF object using lld
+
+    Parameters
+    ----------
+    in_file : str
+        Input file name (relocatable ELF object file)
+
+    out_file : str
+        Output file name (shared ELF object file)
+    """
+    args = "ld.lld -shared " + in_file + " -o " + out_file
```

Comment:
put each argument as an element of the list, and remove shell=True

Reply:
I changed it to `args = ["ld.lld -shared", in_file, " -o ", out_file]` and removed `shell=True`
I am getting the following error:

```shell
Traceback (most recent call last):
  File "/home/aditya/tvm/python/tvm/_ffi/_ctypes/function.py", line 54, in cfun
    rv = local_pyfunc(*pyargs)
  File "/home/aditya/tvm/python/tvm/contrib/rocm.py", line 48, in callback_rocm_link
    rocm_link(tmp_obj, tmp_cobj)
  File "/home/aditya/tvm/python/tvm/contrib/rocm.py", line 21, in rocm_link
    stderr=subprocess.STDOUT)
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory

```

Reply:
Should be  ```["ld.lld", "-shared", in_file, " -o ", out_file]```

Reply:
each argument in its own cell, no space in each option